network/newstream/wire.go

```diff
@@ -111,7 +96,7 @@ type GetRange struct {
 	Ruid      uint
 	Stream    ID
 	From      uint64
-	To        uint64 `rlp:nil`
+	To        *uint64 `rlp:"nil"`
 	BatchSize uint
 	Roundtrip bool
```

Comment:
Please remove `Roundtrip`, I really don't want to have to submit any more PRs like https://github.com/ethersphere/swarm/pull/1422 and https://github.com/ethersphere/swarm/pull/1339

I think developers on the team are smart enough to be able to add this field once they need it.

Reply:
In general do not add code that is not used for no good reason. A good reason is if you will have an immediate PR that will make use of it, for example the PR with `bzz-retrieval`, even though the code was not immediately used is a good example.